services/admin/backend/api/presenter/project.go

```diff
@@ -28,6 +28,36 @@ type Project struct {
 	ShortName   string                 `json:"shortName"`
 	LongName    string                 `json:"longName"`
 	Description string                 `json:"description"`
+	CreatedAt   string                 `json:"createdAt"`
+	CreatedBy   string                 `json:"createdBy"`
+	ChangedAt   string                 `json:"changedAt"`
+	ChangedBy   string                 `json:"changedBy"`
```

Comment:
This is definitely something for later down the line. But seeing `ChangedAt` and `ChangedBy` (I'm guessing that means that we only have the latest change date and user) makes me wonder how we eventually want to be able get the change history of an entity/aggregate. We should discuss this at some point.

Reply:
I think that shouldn't be too hard to do, I can already get the list of events like here:
https://github.com/dasch-swiss/dasch-service-platform/blob/c324a3a43bfbb2d8cd9cfa17252d7494f87e20e2/services/admin/backend/infrastructure/repository/project/project.go#L170

and then I can have a switch statement in Svelte to show a different view or whatever based on the type of event

Reply:
Maybe I'm struggling with the whole event architecture again - but should the frontend really be concerned with the events? I don't think so.  
I was under the impression that the aggregate should be loaded up from the events, but then hold all information in itself - which would then mean, also all the steps of its history - and then the frontend would get a JSON representation of this aggregate. ... or am I mixing layers again?

Reply:
The current `project` entity does have an `changes` field which is an array of the events (history). Currently this isn't being returned in the JSON but I'm sure it could be.

Reply:
I see, you're way ahead of me in that one :)